src/material/standard/shader/textures.glsl

```diff
@@ -102,8 +105,20 @@ vec2 getBaseColorUV()
     vec3 uv = vec3(v_UVCoord1, 1.0);
 #ifdef HAS_BASE_COLOR_MAP
     uv.xy = u_BaseColorUVSet < 1 ? v_UVCoord1 : v_UVCoord2;
+#endif
+#ifdef FLIP_UV_X
```

Comment:
Instead of having flip x/y, can't TextureTransform.scale -1 be used?

Reply:
I've not used TextureTransform.scale before didn't realise we could achieve the same effect this way

Reply:
After reviewing the texturetransform.scale method, I've realised it would have a direct change to the texture's transform itself, making it so that texture UVs would be flipped across all uses of that texture outside of this material

Reply:
Maybe creating the texture transform to be stashed on the material would make more sense than stashing it on the texture itself as it is liable to change